downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc

```diff
@@ -23,6 +23,18 @@ rhel-9-for-x86_64-baseos-rpms                              Red Hat Enterprise Li
 ----
 +
 . Ensure that these repos and only these repos are available to the host OS. If you need to know how to do this use this guide:
-https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/package_manifest/repositories.
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_software_with_the_dnf_tool/assembly_managing-custom-software-repositories_managing-software-with-the-dnf-tool[Chapter 10. Managing custom software repositories Red Hat Enterprise Linux]
 
 . Ensure that the host has DNS configured and can resolve hostnames and IPs using a fully qualified domain name (FQDN). This is essential to ensure services can talk to one another.
+
+.Using unbound DNS
+
+To configure unbound DNS refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/ssembly_setting-up-an-unbound-dns-server_networking-infrastructure-services#proc_configuring-unbound-as-a-caching-dns-server_assembly_setting-up-an-unbound-dns-server[Chapter 2. Setting up an unbound DNS server Red Hat Enterprise Linux 9].
+
+.Using BIND DNS
+
+To configure DNS using BIND refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/assembly_setting-up-and-configuring-a-bind-dns-server_networking-infrastructure-services[Chapter 1. Setting up and configuring a BIND DNS server Red Hat Enterprise Linux 9].
+
+.Optional
+
+To have the installer automatically pick up and apply your AAP subscription manifest license, then use this guide to generate a manifest file which can be downloaded for the installer: link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files[Chapter 2. Obtaining a manifest file Red Hat Ansible Automation Platform 2.].
```

Comment:
Should the "then" be there? There isn't a step before this described in this sentence. I would remove it to make the sentence make grammatical sense, unless I'm misunderstanding something here.

Reply:
Nope. :-)